lib/plugins/create/templates/aws-nodejs-typescript/README.md

```diff
@@ -63,26 +63,33 @@ The project code base is mainly located within the `src` folder. This folder is
 ```
 .
 ├── src
-│   ├── functions            # Lambda configuration and source code folder
+│   ├── functions               # Lambda configuration and source code folder
 │   │   ├── hello
-│   │   │   ├── handler.ts   # `Hello` lambda source code
-│   │   │   ├── index.ts     # `Hello` lambda Serverless configuration
-│   │   │   ├── mock.json    # `Hello` lambda input parameter, if any, for local invocation
-│   │   │   └── schema.ts    # `Hello` lambda input event JSON-Schema
+│   │   │   ├── handler.ts      # `Hello` lambda source code
+│   │   │   ├── index.ts        # `Hello` lambda Serverless configuration
+│   │   │   ├── mock.json       # `Hello` lambda input parameter, if any, for local invocation
+│   │   │   └── schema.ts       # `Hello` lambda input event JSON-Schema
 │   │   │
-│   │   └── index.ts         # Import/export of all lambda configurations
+│   │   └── index.ts            # Import/export of all lambda configurations
 │   │
-│   └── libs                 # Lambda shared code
-│       └── apiGateway.ts    # API Gateway specific helpers
+│   └── libs                    # Lambda shared code
+│       └── apiGateway.ts       # API Gateway specific helpers
+│       └── handlerResolver.ts  # Sharable library for resolving lambda handlers
+│       └── logger.ts           # Lambda middleware
```

Comment:
This line is no longer required

Reply:
good spot, i'd forgotten to replace the `logger.ts` with the existing`lambda.ts`.
done now.